### ISSUE TITLE
ACCOUNT_FORMS setting is dummy

### ISSUE THREAD

**Issue by restless** [pennersr/django-allauth#980]
I'd like to use crispy-forms in my signup and login forms. I'm not quite happy with the `ACCOUNT_SIGNUP_FORM_CLASS` as it takes my class as a base only. So I found `ACCOUNT_FORMS` setting and tried to use it. Simple use like:

`ACCOUNT_FORMS = {'signup': 'path.to.MyForm'}` didn't work - that means that there's no effect of having this setting. As the config param has very little docs (but at least has some) I tried to find it in the code and see how is it supposed to be used but... failed. I did a text search throught the code and it just doesn't exist there.

To summarize:
`ACCOUNT_FORMS` seems to only exist in the `Changelog` and `docs/configuration.rst` files. 
Maybe I miss something here but seems to me that this is just a dummy setting with zero supporting code.

**Comment by pennersr** [comment]
See `allauth.account.app_settings`:

    @property
    def FORMS(self):
        return self._setting('FORMS', {})

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/980
